src/main/java/com/minecolonies/apiimp/initializer/ModBuildingsInitializer.java

```diff
@@ -33,6 +33,13 @@ public static void init(final RegistryEvent.Register<BuildingEntry> event)
                                  .setRegistryName(new ResourceLocation(Constants.MOD_ID, ModBuildings.ARCHERY_ID))
                                  .createBuildingEntry();
 
+        ModBuildings.baker = new BuildingEntry.Builder()
```

Comment:
What, how were those two not here?

Reply:
They’re present as bakery and home, respectively.  This is kinda a hacky workaround, but too many places this code touches straddle between the code-side, desc, blockdesc, and user-side hard coded names.  

I’m open to any smarter or more elegant solution; the duplicate entries just seemed the ones least likely to have a negative impact or confuse actual devs by renaming classes or screwing with internal static strings.

Reply:
Ohh, we shouldn't have duplicate entries at all. I'd prefer renaming the original one then.